crates/ruma-common/tests/events/image.rs

```diff
@@ -206,24 +206,22 @@ fn plain_content_deserialization() {
         ]
     });
 
-    assert_matches!(
-        from_json_value::<ImageEventContent>(json_data)
-            .unwrap(),
-        ImageEventContent { message, file, image, thumbnail, caption: Some(caption), .. }
-        if message.find_plain() == Some("Upload: my_cat.png")
-            && message.find_html().is_none()
-            && file.url == "mxc://notareal.hs/abcdef"
-            && image.width == Some(uint!(668))
-            && image.height.is_none()
-            && thumbnail.is_empty()
-            && caption.find_plain() == Some("Look at my cat!")
-    );
+    let content = from_json_value::<ImageEventContent>(json_data).unwrap();
+    assert_eq!(content.message.find_plain(), Some("Upload: my_cat.png"));
+    assert_eq!(content.message.find_html(), None);
+    assert_eq!(content.file.url, "mxc://notareal.hs/abcdef");
+    assert_matches!(content.file.encryption_info, None);
+    assert_eq!(content.image.width, Some(uint!(668)));
+    assert_eq!(content.image.height, None);
+    assert!(content.thumbnail.is_empty());
+    let caption = content.caption.unwrap();
```

Comment:
```suggestion
    let caption = assert_matches!(content.caption, Some(c) => c);
```

Reply:
@zecakeh wdyt about this? I'm slightly in favor of `.unwrap()`, just because it's shorter and much easier to type for new tests.

Reply:
I'm also in favor of `.unwrap()` for the same reason.

The goal here is to have more detailed error messages to know quicker where there's an issue. The `assert_matches` form won't give more details.

Reply:
hm, alright, then this should be fine still :+1: